### PR TITLE
Add bitcoincashd cryptocurrency chart

### PR DESCRIPTION
**Pull request by govale**
*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

### CODE REVIEW

File: incubator/bitcoincashd/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the bitcoincashd chart and their default values.
+
+Parameter                  | Description                        | Default                                                   
+-----------------------    | ---------------------------------- | ----------------------------------------------------------
+`image.repository`         | Image source repository name       | `arilot/docker-bitcoincashd`
+`imageTag`                 | `bitcoincashd` release tag.        | `1.0`
```
Comment (unguiculus) [inline incubator/bitcoincashd/README.md]: image.tag

File: incubator/bitcoincashd/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+`imagePullPolicy`          | Image pull policy                  | `IfNotPresent`
```
Comment (unguiculus) [inline incubator/bitcoincashd/README.md]: image.pullPolicy

File: incubator/bitcoincashd/templates/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "bitcoincashd.fullname" . }}
+  labels:
+    app: {{ template "bitcoincashd.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```
Comment (unguiculus) [inline incubator/bitcoincashd/templates/configmap.yaml]: Update `chart` labels: `chart: {{ template "bitcoincashd.chart" . }}`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/bitcoincashd/README.md (hunk 1 above)
Comment (govale) [reply incubator/bitcoincashd/README.md]: Fixed.

File: incubator/bitcoincashd/README.md (hunk 2 above)
Comment (govale) [reply incubator/bitcoincashd/README.md]: Fixed.

File: incubator/bitcoincashd/templates/configmap.yaml (hunk 3 above)
Comment (govale) [reply incubator/bitcoincashd/templates/configmap.yaml]: Fixed.

File: incubator/bitcoincashd/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      {{- end }}
+      containers:
+        - name: {{ template "bitcoincashd.fullname" . }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          resources:
+  {{ toYaml .Values.resources | indent 10 }}
+          ports:
+            - name: rpc
+              containerPort: {{ .Values.service.rpcPort }}
```
Comment (unguiculus) [inline incubator/bitcoincashd/templates/deployment.yaml:40]: It probably doesn't make sense to have these configurable like this. Changing the service ports also changes container ports which is likely to break things. I'd suggest you hard-code container ports.

File: incubator/bitcoincashd/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+              mountPath: /configmap
+            - name: config
+              mountPath: /root/.bitcoin/
+      {{- end }}
+      containers:
+        - name: {{ template "bitcoincashd.fullname" . }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          resources:
+  {{ toYaml .Values.resources | indent 10 }}
```
Comment (unguiculus) [inline incubator/bitcoincashd/templates/deployment.yaml:37]: `{{ toYaml .Values.resources | indent 12 }}`

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Pull request closed without merging**